concent_api/core/tests/test_auth_report_computed_task.py

```diff
@@ -920,16 +920,6 @@ def test_provider_forces_computed_task_report_and_requestor_sends_rejection_due_
         self.assertEqual(response.status_code,        204)
         self.assertEqual(len(response.content),       0)
 
-        with freeze_time("2017-12-01 11:00:15"):
```

Comment:
Why is this removed ?

Reply:
Because this part return "out-of-band" message that we check later, so I don't think we need to replace it with different call.

Reply:
In other words, I think it should stay it if doesn't break anything and test one more corner case.

Reply:
Okey, I'll switch it to other case.

Reply:
As discussed, it is ok to remove this as it is doing the same now as `# Step 9` of this test.